en/includes/guides/flows/build-a-flow.md

```diff
@@ -1,90 +1,113 @@
+#### Generate with AI
 
 With **Generate with AI**, you can use plain language instructions to build a complete flow, allowing you to start with a minimal effort and customize components as needed.
 
 ![Flow builder AI]({{base_path}}/assets/img/guides/flows/flow-builder-generate-with-ai.png){: width="auto" style="display: block; margin: 0;"}
 
 {% endif %}
 
-### Use a starter template
-
-Starter templates offer commonly used starting points, so you don’t have to begin from scratch. Click the `+` button next to a template to add it to the flow.
+#### Use a starter template
 
-![Starter Templates]({{base_path}}/assets/img/guides/flows/flow-builder-starter-templates.png){: width="auto" style="display: block; margin: 0;"}
+Select from a list of pre-built templates for common use cases to avoid starting from scratch. Click the `+` icon next to a template to add it to the canvas.
 
-### Build from scratch
+![Starter Templates]({{base_path}}/assets/img/guides/flows/flow-password-recovery-start-from-template.png){: width="auto" style="display: block; margin: 0;"}
 
-If you prefer to take complete control of the building process, you can start with a blank canvas and add steps, widgets, and components to build your flow.
+#### Build from scratch
 
-![Custom Flow]({{base_path}}/assets/img/guides/flows/flow-builder-custom-flow.png){: width="auto" style="display: block; margin: 0;"}
+For complete control, start with a blank canvas and build the flow by dragging and dropping the required [flow elements](#flow-elements).
 
-## Enable a flow
+![Build flow manually]({{base_path}}/assets/img/guides/flows/flow-builder-build-flow-manually.png){: width="auto" style="display: block; margin: 0;"}
 
-{{product_name}} provides default sequences for every flow in the flow builder. You can design and refine your customized flow at your own pace, keeping it disabled until you’re ready to share it. While your customized flow is disabled, users continue to experience the default flow. Once enabled, your users will seamlessly change to the customized experience you’ve created.
+!!! Note
 
-To enable a flow,
+    Save the flow at any point while building. The saved version stays as a draft. Edit the draft later as needed.
 
-1. On the {{product_name}} Console, go to **Flows**.
+### Step 3: Enable the flow
 
-2. Select the flow you want to enable.
+Once the prepared flow is ready to publish for users, enable it.
 
-3. On the top-right corner of the screen, turn the toggle ON to enable the flow.
+1. On the top-right corner of the screen, turn the toggle ON to enable the flow.
 
+!!! warning "Immediate Publication"
+    Enabling the flow will immediately publish the current configuration on the canvas, making it live for all users.
 !!! Note
 
     If you disable a customized flow, your users will automatically continue with the default experience until you’re ready to publish it again.
 
-## Reference: Flow elements
+## Flow inheritance for sub organizations
```

Comment:
This needs to move from this doc to the organizations section. We have a separate section in Organizations to explain how inheritance work. This info is irrelevant to the flow doc.

Reply:
https://is.docs.wso2.com/en/next/guides/organization-management/inheritance-in-organizations/